core/pom.xml

```
@@ -46,15 +46,15 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>fish.payara.server.core</groupId>
     <artifactId>core-aggregator</artifactId>
-    <version>7.0.0-SNAPSHOT</version>
+    <version>7.0.0.Alpha2-SNAPSHOT</version>
 
     <packaging>pom</packaging>
     <name>Payara Core Aggregator</name>
 
     <description>All of Payara Core modules</description>
 
     <properties>
-        <payara.core.version>7.0.0-SNAPSHOT</payara.core.version>
+        <payara.core.version>7.0.0.Alpha2-SNAPSHOT</payara.core.version>
         <payara.core.compare-version>6.13.0</payara.core.compare-version>
```

Should be updated to released core.
```suggestion
        <payara.core.compare-version>7.0.0.Alpha1</payara.core.compare-version>
```